tools/bench/sim/split.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+#
+# formatted with black
+import os
+os.environ["OMP_NUM_THREADS"] = "1" # export OMP_NUM_THREADS=1
```

Comment:
could you comment a bit on why we have to set these env var?

Reply:
done